components/blitz/src/omero/gateway/facility/ROIFacility.java

```diff
@@ -43,7 +43,6 @@
 import omero.gateway.model.ROIResult;
 import omero.gateway.util.ModelMapper;
 import omero.gateway.util.PyTablesUtils;
-import omero.model.IObject;
```

Comment:
@dominikl: can we leave this `import` cleanup out of this PR? As far as I can tell, this is the only place that would remain in conflict with `regions`.